mail_full_expand/views/mail_full_expand.xml

```diff
@@ -3,44 +3,38 @@
 <!-- © 2014-2015 Grupo ESOC <http://www.grupoesoc.es>
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
 
-<openerp>
-<data>
-
-<record id="view" model="ir.ui.view">
-    <field name="name">mail.message.full_expand</field>
-    <field name="model">mail.message</field>
-    <field name="arch" type="xml">
-        <form string="Message" version="7.0">
-            <group>
-                <field name="author_id" readonly="1"/>
-                <field name="email_from" readonly="1"/>
-                <field name="date" readonly="1"/>
-                <field name="partner_ids"
-                       widget="many2many_tags"
-                       readonly="1"/>
-                <field name="notified_partner_ids"
-                       widget="many2many_tags"
+<odoo>
+    <record id="view" model="ir.ui.view">
+        <field name="name">mail.message.full_expand</field>
+        <field name="model">mail.message</field>
+        <field name="priority">30</field>
```

Comment:
Yes, otherwise the action on the "mail messages" menuitem in technical settings will load this view instead of the "standard" one (`mail.view_message_form`) which has, for some unfathomable reason, a priority of 20 instead of the 16 default...